src/app/shared/libraries/accordion-item/accordion-item.component.html

```diff
@@ -0,0 +1,15 @@
+
+<div class="govuk-accordion__section">
+  <div class="govuk-accordion__section-header">
+    <h2 class="govuk-accordion__section-heading">
+      <span class="govuk-link govuk-!-font-size-36">{{title}}</span>
+      <span class="govuk-accordion__section-button" id="accordion-default-heading"></span>
+    </h2>
+  </div>
+  <div id="accordion-default-content" class="govuk-accordion__section-content">
```

Comment:
i assume we need an id for enabling the expand/collapse logic. This id seems quite generic, can we have a naming more unique assuming we can have multiple please? We might need to dynamically create the id if it is the case, it could be the section name for example `accordion-default-content${titleSection}` (titleSection will be the unique name/section of the form)